src/gatsby-node.js

```diff
@@ -0,0 +1,577 @@
+      `find public .cache  -type f -iname "*.bmp" -or -iname "*.tif" -or -iname "*.webp" -or -iname "*.svg" | wc -l`
+    );
+    const benchmarkMetadata = this.getMetadata();
+    const attributes = {
+      ...ciAttributes,
+      ...benchmarkMetadata,
+      ...metrics.tags,
+      gatsbySite: PLUGIN_OPTIONS.SITE_NAME,
+      buildId: PLUGIN_OPTIONS.buildId,
+      gitHash,
+      gitAuthor,
+      gitCommitTimestamp,
+      gitRepoName,
+      ciName: CI_NAME,
+      nodeEnv,
+      newRelicSiteName: PLUGIN_OPTIONS.SITE_NAME,
+      nodejs: nodejsVersion,
+      gatsby: gatsbyVersion,
+      gatsbyCli: gatsbyCliVersion,
+      sharp: sharpVersion,
+      webpack: webpackVersion,
+      coreCount: coreCount,
+      deletedPages: DELETED_PAGES,
+      changedPages: CHANGED_PAGES,
+      clearedCache: CLEARING_CACHE,
+    };
+    const buildtimes = {
+      ...attributes,
+      bootstrapTime: this.timestamps.bootstrapTime,
+      instantiationTime: this.timestamps.instantiationTime,
+      // Instantiation time of this class
+      benchmarkStart: this.timestamps.benchmarkStart,
+      // Start of benchmark itself
+      preInit: this.timestamps.preInit,
+      // Gatsby onPreInit life cycle
+      preBootstrap: this.timestamps.preBootstrap,
+      // Gatsby onPreBootstrap life cycle
+      preBuild: this.timestamps.preBuild,
+      // Gatsby onPreBuild life cycle
+      postBuild: this.timestamps.postBuild,
+      // Gatsby onPostBuild life cycle
+      benchmarkEnd: this.timestamps.benchmarkEnd, // End of benchmark itself
+    };
+    var timestamp = Date.now();
+    const timeelapsed =
+      this.timestamps.benchmarkEnd - this.timestamps.benchmarkStart;
+    return [
+      {
+        metrics: [
+          {
+            name: "mdxFiles",
+            type: "gauge",
+            value: mdxCount,
+            timestamp: timestamp,
+            attributes: attributes,
+          },
+          {
+            name: "jsSize",
+            type: "gauge",
+            value: publicJsSize,
+            timestamp: timestamp,
+            attributes: attributes,
+          },
+          {
+            name: "pngs",
+            type: "gauge",
+            value: pngCount,
+            timestamp: timestamp,
+            attributes: attributes,
+          },
+          {
+            name: "jpgs",
+            type: "gauge",
+            value: jpgCount,
+            timestamp: timestamp,
+            attributes: attributes,
+          },
+          {
+            name: "otherImages",
+            type: "gauge",
+            value: otherImagesCount,
+            timestamp: timestamp,
+            attributes: attributes,
+          },
+          {
+            name: "gifs",
+            type: "gauge",
+            value: gifCount,
+            timestamp: timestamp,
+            attributes: attributes,
+          },
+          {
+            name: "memory-rss",
+            type: "gauge",
+            value: rss !== null && rss !== void 0 ? rss : 0,
```

Comment:
Could be simplified to the following (since `null` and `void` are falsy):

```suggestion
            value: rss ? rss : 0,
```

Reply:
Same with the other `null` / `void` checks.